src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -4,6 +4,14 @@
 
 	private int[] NUMBERS = new int[12];
 
+	public int getTotal() {

```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.